test/support/test_helpers.ex

```diff
@@ -1,15 +1,33 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.Repo
+  alias RemoteRetro.{Repo, User}
 
   def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro} = context
-    idea = Map.put(idea, :retro_id, retro.id)
+    %{idea: idea, retro: retro, user: user} = context
+    user =
+      if Map.has_key?(user, :id) do
```

Comment:
The problem was that the user needed to be in the test database in order to not get a foreign key constraint error, so adding an ID to the mock user didn't work because the mock_user was not persisted in the test database for the unit tests. However it was in the DB for the e2e tests, so I figured for those tests we should just use the user that's already there instead of persisting another one. It does lead to this clunky code though. I'll change to persisting a new user in the database for the e2e tests.